.github/workflows/push-dev-docker-images.yml

```diff
@@ -79,4 +79,15 @@ jobs:
           GIT_COMMIT=${{ github.sha }}
         tags: |
           ${{ env.OSMOSIS_DEV_IMAGE_REPOSITORY }}:${{ env.DOCKER_IMAGE_TAG }}
-          ${{ env.OSMOSIS_INIT_CHAIN_IMAGE_REPOSITORY }}:${{ env.DOCKER_IMAGE_TAG }}
```

Comment:
Note: this removed line was essentially pushing a regular Osmosis image in the wrong place: https://hub.docker.com/r/osmolabs/osmosis-e2e-init-chain/tags